action.yml

```diff
@@ -117,7 +117,6 @@ runs:
       id: setup-python
       with:
         python-version: '3.11'
-        update-environment: false
```

Comment:
These changes are undesired, unless you can tell me why you want to use the same venv this action uses instead of setting up your own venv.